### PR TITLE
fix(onboard step): [MAG-3448] update option for grey background colour

### PR DESCRIPTION
**Pull request by eTheMan80**
# <JIRA issue / Title>

## TL;DR

-   update option for grey background colour
-  
-  
-

## Details

Checklist / Reminders:

-   [x] Documentation / Story is created / adapted
-   [ ] Unit tests
-   [ ] CSS changes are BEM-compliant

### CODE REVIEW

**Review by tricaalexandru — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -1345,13 +1345,6 @@
   resolved "https://registry.yarnpkg.com/@emotion/weak-memoize/-/weak-memoize-0.2.5.tgz#8eed982e2ee6f7f4e44c253e12962980791efd46"
   integrity sha512-6U71C2Wp7r5XtFtQzYrW5iKFT67OixrSxjI4MptCHzdSVlgabczzqLe0ZSgnub/5Kp4hSbpDB1tMytZY9pwxxA==
 
-"@excelwithbusiness/webmobile-svg-library@^0.0.14":
```
Comment (tricaalexandru) [inline yarn.lock:1348]: @eTheMan80 is this supposed to be removed?

Comment (eTheMan80) [reply yarn.lock:1348]: @tricaalexandru @matthill8286  I ran `yarn` and the package was removed.

**Comment by matthill8286** [comment]
@eTheMan80 @tricaalexandru Whats this for?? the background color??

File: yarn.lock (hunk 1 above)
Comment (matthill8286) [reply yarn.lock:1348]: @tricaalexandru correct it's no longer a dependency as it's now linked.

**Comment by matthill8286** [comment]
@eTheMan80 @tricaalexandru Can you please add screenshots as i've no idea what this is for, and from what I see you've changed a component thats not used in onboarding.

**Comment by eTheMan80** [comment]
@matthill8286 I've added a prop to be able to set the background colour of the stepper component on onboarding. I can include a screenshot of what this will look like but won't commit the code because I need to pass `surface="grey4"` in OnboardingSteps component.

**Comment by eTheMan80** [comment]
@matthill8286 the Steps component uses the Card component `<Card
      elevation={elevation}
      noBorder={['top', 'right', 'bottom', 'left']}
      padding="md"
      {...other}>
      <FlexBox display="flex" alignItems="center" flexDirection="row">
        {steps}
      </FlexBox>
    </Card>`

**Review by philjonasfiltered — APPROVED** [review]

**Pull request merged**